tests/plugins/test_time_slider_choropleth.py

```diff
@@ -51,23 +61,21 @@ def norm(col):
 
     m = folium.Map((0, 0), tiles='Stamen Watercolor', zoom_start=2)
 
-    folium.plugins.TimeDynamicGeoJson(
+    time_slider_choropleth = TimeSliderChoropleth(
         gdf.to_json(),
         styledict
-    ).add_to(m)
+    )
+    time_slider_choropleth.add_to(m)
 
-    m._repr_html_()
+    rendered = time_slider_choropleth._template.module.script(time_slider_choropleth)
 
+    m._repr_html_()
     out = m._parent.render()
+    assert '<script src="https://d3js.org/d3.v4.min.js"></script>' in out
 
-    # We verify that imports
-    assert '<script src="https://d3js.org/d3.v4.min.js' in out  # noqa
-
-    # We verify that data has been inserted currectly
-    expected_timestamps = ("var timestamps = ['1454166000', '1456671600', '1459350000', "
-                           "'1461942000', '1464620400', '1467212400', '1469890800', "
-                           "'1472569200', '1475161200', '1477839600'];")
-    #assert expected_timestamps in out
+    # We verify that data has been inserted correctly
+    expected_timestamps = """var timestamps = ["1454198400", "1456704000", "1459382400"];"""  # noqa
```

Comment:
I reduced this to make it easier to compare.
Not to self: these values are not the same on Travis-CI and locally. I need to double check this!